src/api/lib/tasks/databases.rake

```diff
@@ -83,6 +83,20 @@ namespace :db do
       end
       File.open("#{Rails.root}/db/structure.sql", "w+") { |f| f << new_structure }
     end
+
+    desc "Verify that structure.sql in git is up to date"
+    task verify: :environment do
+      puts "Running rails db:migrate"
+      Rake::Task["db:migrate"].invoke
```

Comment:
In the test suite this will be the case. And if someone runs it locally, I wouldn't want to wipe his db.

Besides, why would the db have to be empty? The structure sql reflects the, well, structure of the  db, not it's content.